### PR TITLE
Add beian option at footer for Chinese website owner

### PR DESCRIPTION
**Pull request by sli1989**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [ ] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [ ] Tests for the changes have been added (for bug fixes / features).
   - [ ] Muse | Mist have been tested.
   - [ ] Pisces | Gemini have been tested.
- [ ] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [ ] Bugfix.
- [x] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number(s): N/A

## What is the new behavior?
ADD: `beian` option in ` _config.yml` - `theme` - `footer`.

* Screens with this changes: 
* Link to demo site with this changes: N/A

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [ ] No.

### CODE REVIEW

File: layout/_partials/footer.swig [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <div class="copyright">{#
 #}{% set current = date(Date.now(), "YYYY") %}{#
-#}&copy; {% if theme.footer.since and theme.footer.since != current %}{{ theme.footer.since }} – {% endif %}{#
```
Comment (ivan-nginx) [inline layout/_partials/footer.swig]: Here is not `&mdash;`. Here is [n-dash](https://www.punctuationmatters.com/hyphen-dash-n-dash-and-m-dash/) (this: –, not that: —)

Comment (sli1989) [reply layout/_partials/footer.swig]: ok, `&mdash`  will be displaced.

File: layout/_partials/footer.swig [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+#}{% if theme.footer.beian.enable %}{#
+#}<a target="_blank" rel="external nofollow" href="http://www.miitbeian.gov.cn/"> {{theme.footer.beian.icp}} </a>{#
```
Comment (MoshiYeap) [inline layout/_partials/footer.swig]: I opened the "beian" option today and found that the "beian" option and "since2015" are crowded in one line. It is not very beautiful. Can I make the "beian" information text appear on a separate line?

Comment (ivan-nginx) [reply layout/_partials/footer.swig]: You can create your own pull request with this changes. Welcome!

Comment (MoshiYeap) [reply layout/_partials/footer.swig]: > You can create your own pull request with this changes. Welcomeo'k!
ok，thanks reply。

**Pull request merged**